src/site/content/en/blog/shape-detection/index.md

```diff
@@ -116,16 +117,16 @@ of use cases for all three features.
 | 2. Create initial draft of specification   | [In Progress][spec]          |
 | **3. Gather feedback & iterate on design** | [**In progress**](#feedback) |
 | 4. Origin trial                            | [Complete](https://developers.chrome.com/origintrials/#/view_trial/-2341871806232657919) |
-| 5. Launch                                  | Not started                  |
+| **5. Launch**                              | **Complete** (Barcode detection) |
 
 </div>
 
 ## How to use the Shape Detection API {: #use }
 
 {% Aside 'warning' %}
-  The origin trial ended in Chrome 78 and the API will be
-  turned off while the Chrome team integrates developer feedback. You can
-  always use the Shape Detection API for local experiments by enabling the
+  So far only barcode detection has launched in Chrome 82,
```

Comment:
```suggestion
  So far only barcode detection is available by default, starting in Chrome 82.
```

Reply:
The old version makes it sound as though face and shape detection will eventually be available in 82.